frame/example/src/lib.rs

```diff
@@ -130,14 +132,15 @@
 //!
 //! <!-- Original author of paragraph: ??? -->
 //!
-//! // Reference documentation of aspects such as `storageItems` and `dispatchable` functions should only be
-//! // included in the <https://docs.rs> Rustdocs for Substrate and not repeated in the README file.
+//! // Reference documentation of aspects such as `storageItems` and `dispatchable` functions should
+//! only be // included in the <https://docs.rs> Rustdocs for Substrate and not repeated in the README file.
```

Comment:
What's with the `//! //`s here?

Reply:
it is because the previous doc was doing `//! //`, which is quite confusing to rustfmt.